---
title: "Introduction"
format:
  html:
    toc: false
    number-sections: false
---

`scMarco` [@Chen2023.02.03.527019] aims to streamline the selection of actionable marker combinations with a friendly exploration interface powered by a Bayesian mixture model [@davis2020; @Ã¶zel2021] to binarize gene expression in scRNA-seq.

`scMarco` can be run locally on your personal computer if you have `R` and supporting packages including `Shiny` installed. It can also be deployed on the internet like any Shiny app if you wish to provide your datasets to the public.

To prepare your data for `scMarco`, we provide a `Snakemake` workflow to discover and binarize genes that are bimodal at per-cluster level, and [instructions to convert binarized and normalized expression matrices](preprocess.qmd) to be imported to `scMarco`.

Once `scMarco` is [configured](config.qmd), we recommend users to start **from a cluster (From cluster)** or **from a gene that marks several clusters (From gene)** of interest. Either tab will help you identify other clusters that could be marked when you only use one marker, and help you select another marker to specifically label it (**Find distinct genes within**).

Once marker combinations are found, you might want to see how other clusters in the same dataset express this combination (**Co-exprssion plot**).

We acknowledge that while binarization greatly reduces computational complexity, it has its limitation: Gene expression is a spectrum, and there would be clusters that are hard to categorize to either ON or OFF state. We thus encourage users to also examine log-normalized expression value and its dynamic (**Plot expression trend**) and decide whether a satisfactory pair of markers have been identified.

## Contact

If you have any questions, please contact [Yu-Chieh David Chen](mailto:ycc4@nyu.edu)
about suggestions and inquiries about fly lines and genetics; if you have
questions, suggestions, or bugs noted about `scMarco`, please contact
[Yen-Chung Chen](mailto:yenchung.chen@nyu.edu).

## References
